Build/Laptop/EMail.cc

```diff
@@ -1916,7 +1917,7 @@ static void HandleIMPCharProfileResultsMessage(void)
 		// have to place players name into string for first record
 		if (i == 0)
 		{
-			wchar_t zTemp[512];
+			wchar_t zTemp[NAME_LENGTH];
```

Comment:
no, that the destination in the following copy (wcscat) is smaller and could be overrun.